lib/browser/guest-view-manager.ts

```diff
@@ -46,23 +60,24 @@ const supportedWebViewEvents = [
   'update-target-url'
 ];
 
-const guestInstances = {};
-const embedderElementsMap = {};
+const guestInstances: { [key: string]: WebContents & Electron.WebContentsInternal | any; } = {};
+const embedderElementsMap: { [key: string]: number } = {};
 
-function sanitizeOptionsForGuest (options) {
+function sanitizeOptionsForGuest (options: BrowserWindow) {
   const ret = { ...options };
   // WebContents values can't be sent over IPC.
   delete ret.webContents;
   return ret;
 }
 
 // Create a new guest instance.
-const createGuest = function (embedder, params) {
+const createGuest = function (embedder: Electron.WebContentsInternal & WebContents, params: Record<string, any>): number {
   if (webViewManager == null) {
     webViewManager = process._linkedBinding('electron_browser_web_view_manager');
   }
 
-  const guest = webContents.create({
+  // TS saying .create doesn't exist, could not find it as I walked back the dep tree.
+  const guest = (webContents as any).create({
```

Comment:
This is a private internal method so it should be declared in `internal-electron.d.ts`.  The syntax is:

```
create(opts: Electron.WebPreferences): Electron.WebContents
```

Reply:
Appreciate that, this is the one that had me the most confused.

Reply:
hmm, adding this to the typings in that file still does not resolve the issue where .create does not exist.